tendrl/notifier/utils/central_store_util.py

```diff
@@ -0,0 +1,69 @@
+from etcd import EtcdException
+from etcd import EtcdKeyNotFound
+from tendrl.notifier.objects.alert import Alert
+from tendrl.notifier.objects.cluster_alert import ClusterAlert
+from tendrl.notifier.objects.node_alert import NodeAlert
+
+
+def read_key(key):
```

Comment:
Probably this you can use the one from commons commons has a function similar to this